dcpy/connectors/edm/publishing.py

```diff
@@ -37,12 +37,18 @@ def get_build_metadata(product_key: ProductKey) -> BuildMetadata:
     return BuildMetadata(**yaml.safe_load(file_content))
 
 
-def get_version(product_key: ProductKey) -> str:
-    """Given product key, gets version"""
-    return get_build_metadata(product_key).version
+def get_version(product_key: ProductKey) -> str | None:
```

Comment:
And then [this](https://github.com/NYCPlanning/data-engineering/pull/975/commits/dec89121d821ad4271e9062ad002a68b0fcfb0ff#diff-c39cbd5c9784cd5e96cca1ad8aa91e9b20a8e76b0d6b3ad43d47a0d67e07c717R219) isn't needed - which makes sense, since we fully expect to get a version there